pepper-apis/src/main/java/org/broadinstitute/ddp/route/UserRegistrationRoute.java

```diff
@@ -580,6 +580,7 @@ private void initializeProfile(Handle handle, User user, UserRegistrationPayload
                     .setLastName(lastName)
                     .setPreferredLangId(languageId)
                     .setTimeZone(timeZone)
+                    .setSkipLanguagePopup(false)
```

Comment:
Is it possible to leave this `null` to indicate that user has not explicitly made a choice? Or will `null` break other things that's reading this flag?